[torch] Add fmsub to vectrozation primitives

efiks opened the pull request:
Summary: Add fmsub  which is similar to fmadd

Test Plan: CI

Differential Revision: D40215267

efiks commented:
/easycla

efiks commented:
/easycla

efiks commented:
/easycla

efiks commented:
/easycla

ajtulloch commented:
For completeness can you add the bfloat16 variant as well following the bfloat16 fmadd example?

efiks committed:
[torch] Add fmsub to vectrozation primitives (#86568)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/86568

Add fmsub  which is similar to fmadd

Test Plan: CI

Reviewed By: ajtulloch

Differential Revision: D40215267

fbshipit-source-id: b2c2f680aec8e1ec3d48dd3b5b4519cf00d97d8a

efiks commented:
@pytorchbot merge

efiks commented:
@pytorchbot merge -f

efiks commented:
@pytorchbot merge -f "flaky test"

malfet commented:
@pytorchbot merge -f "MPS failures seems to be unrelated"
